packages/jest-environment-node/src/index.js

```diff
@@ -37,9 +37,13 @@ class NodeEnvironment {
     global.global = global;
     global.clearInterval = clearInterval;
     global.clearTimeout = clearTimeout;
-    global.Promise = Promise;
     global.setInterval = setInterval;
     global.setTimeout = setTimeout;
+    // URL and URLSearchParams are global in Node >= 10
+    if(typeof URL !== 'undefined' && typeof URLSearchParams !== 'undefined') {
```

Comment:
Mind fixing the linting error?

Needs to tell eslint to ignore globals. Potentially update `globals` package if it's within semver range (https://github.com/sindresorhus/globals/commit/0a57eb52ff7a1c92f227d2441349e0441252e8bb)